sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -1275,7 +1275,7 @@ ALTER TABLE `form_observation` ADD `ob_reason_status` VARCHAR(32) NULL;
 ALTER TABLE `form_observation` ADD `ob_reason_code` VARCHAR(32) NULL;
 #EndIf
 #IfMissingColumn form_observation ob_reason_text
-ALTER TABLE `form_observation` ADD `ob_reason_text` TEXT;
+ALTER TABLE `form_observation` ADD `ob_reason_text` TEXT NULL;
```

Comment:
this is conflict and can be removed from PR as current master is correct. Same for database below.